### ISSUE TITLE
[IOS] The entire ListView move instead of one item when doing a drag and drop.

### ISSUE THREAD

**Issue by Marc-Antoine-Soucy** [unoplatform/uno#12207]
### Current behavior

When you try to move an item in a listview, sometimes, or always, depending on the container style, the entire listview will move along with the item.

### Expected behavior

Just like on Android and WinUI, only the selected item must move.

### How to reproduce it (as minimally and precisely as possible)

I will make a sample later.

### Workaround

In the ItemContainerStyle template, you need to add an invisible rectangle (or any other UI control) whose height is larger than the expected height of the ContentPresenter.

### Works on UWP/WinUI

Yes

### Environment

Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia

### NuGet package version(s)

4.9.0-dev.861
and 4.8.33

### Affected platforms

iOS

### IDE

Visual Studio 2022

### IDE version

Version 17.4.4

**Comment by Marc-Antoine-Soucy** [comment]
I will add a sample and gifs later, but I just added this for now in case someone would need the workaround.

**Comment by Marc-Antoine-Soucy** [comment]
This is might be the same bug I misidentified in this issue? https://github.com/unoplatform/uno.extensions/issues/997

**Comment by dr1rrb** [comment]
Yes this should have been fixed by https://github.com/unoplatform/uno/pull/11529. Could you please validate the version of  uno used by your app (the safest way to validate it is by adding a `<TextBlock x:Name="_unoVersion" />` in your page, then in the ctor of your page, in the code behind, add this `_unoVersion.Text = typeof(UIElement).Assembly.GetCustomAttributesData().FirstOrDefault(data => data.AttributeType.Name.Contains("AssemblyInformationalVersion", StringComparison.OrdinalIgnoreCase))?.ConstructorArguments.FirstOrDefault().Value?.ToString();` )

**Comment by Marc-Antoine-Soucy** [comment]
I will check this out, but I very much doubt that I didnt upgrade the packages correctly, since updating them had visible side effects like giving drag and drop actual animations that they didnt have before updating them.
